android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -170,6 +170,14 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
                 isPersonalizedPrice = call.argument("isPersonalizedPrice"); // TODO: Rename
                 purchasePackage(packageIdentifier, offeringIdentifier, oldSKU, prorationMode, isPersonalizedPrice, result);
                 break;
+            case "purchaseSubscriptionOption":
+                productIdentifier = call.argument("productIdentifier");
```

Comment:
Hmm this drove me crazy... It seems we are sharing the variable declaration between cases. That's not great. Still no need to change it in this PR (it was like this before)